skyportal/handlers/api/observation_plan.py

```diff
@@ -2078,7 +2078,7 @@ def observation_simsurvey(
         Optional parameters to specify the injection type, along with a list of possible values (to be used in a dropdown UI)
     """
 
-    session = Session()
+    session = Session(bind=DBSession.session_factory.kw["bind"])
```

Comment:
It seems very surprising to me that there would be a difference between setting the `bind` at the `sessionmaker()` level vs. the `Session()` level. Is there any reason given for why this has to be this way in 2.0?

Reply:
best I can tell they are making it more explicit, and our out of session calls (async calls) requires this).